language/tools/move-cli/README.md

```diff
@@ -181,6 +181,9 @@ happening):
 $ move sandbox publish -v
 Compiling Move modules...
 Found and compiled 1 modules
+Warning: Found script in specified files for publishing. But scripts cannot be published. Script found in: src/scripts/debug_script.move
```

Comment:
I think at this point, there should be no `debug_script.move` right?  This `Warning` message should not exist.

Reply:
Ah, I see, that is right. It is there. @tzakian for comment on whether we are OK with the warning here.

Reply:
Having a warning in the README (especially in the section on how to structure things) doesn't seem right to me. 

I think it might be worth restructuring the project layout description so that the scripts aren't under `src` anymore (and modules are just directly under `src` and not `src/modules`).

Reply:
That sounds good! I'll work on updating it.

Reply:
@tzakian I've realized that this could require changing the behavior of the `move sandbox test` `--create` flag, as it uses the same directory structure as the examples. Are you ok with that?

With my change the new directory structure created by `move sandbox test new_test --create` will be:
```
new_test
├── args.txt
├── scripts
└── src
```
I decided to leave a `scripts` directory as it seemed to be cleaner (and it doesn't trigger the previous warning), but I can remove it if you'd like.

Reply:
This looks great to me! I think keeping the scripts directory makes sense and is cleaner, so would be in favor of keeping it as well.

Reply:
It's somewhat outside of the scope of this PR, but I think ideally the README should show a new dev how to call a script function, since that's likely to be more useful...

Reply:
@sblackshear Currently there's an example where a function defined in a newly created module is called in a script (in the "Updating state" section). Are you saying that you also want an example where a function defined in one script is called in another script?

Reply:
My understanding of what @sblackshear means is: the CLI can run either
1. a `Script`, via `move sandbox run <path-to-script> <args...>` or
2. a `public(script) fun` defined in some module, via `move sandbox run <module-path> <function-name> <args...>`.

The current tutorial shows how to do 1), but since `Script` is rarely used in Diem now and `ScriptFunction` is the preferred way of designating an execution entry point. It would be good to demonstrate 2) in the tutorial.

But this is likely outside of the scope of this PR.

Reply:
I see, I will leave it out of this PR then.